routes.yaml

```diff
@@ -0,0 +1,5 @@
+app_name: acsp-profile-delta-consumer
```

Comment:
routes.yaml was a requirement for Mesos services so can be removed

Reply:
removed and ticket updated